ghost/email-service/i18n-setup.js

```diff
@@ -0,0 +1,15 @@
+const i18nLib = require('../i18n');
+const settingsCache = require('../core/core/shared/settings-cache');
```

Comment:
You might need to make a part of the email renderer since I think the `settingsCache` needs to be provided as a dependency rather than imported